pkg/config/config_windows.go

```diff
@@ -8,7 +8,7 @@ package config
 const (
 	defaultConfdPath            = "c:\\programdata\\datadog\\conf.d"
 	defaultAdditionalChecksPath = "c:\\programdata\\datadog\\checks.d"
-	defaultRunPath              = ""
+	defaultRunPath              = "c:\\programdata\\datadog\\run"
```

Comment:
The alternative would be to not make changes to the install at all, and just put it at the root
c:\\programdata\\datadog would be fine. But that's up to you